how-to-use-azureml/machine-learning-pipelines/intro-to-pipelines/aml-pipelines-schedule-with-pipeline-parameter-built-in-functions.ipynb

```diff
@@ -0,0 +1,392 @@
+    "See below table for supported built-in functions in schedule.\n",
+    "\n",
+    "| Function    | Description |\n",
+    "| :---------- | :---------- |\n",
+    "| `trigger().scheduledTime` | Return time at which the trigger was scheduled to invoke the pipeline run. Consistent with the time you specific in `ScheduleRecurrence` instance. This expression can only be resolved in schedule scenario.\n",
+    "| `trigger().startTime` | Return time at which the trigger actually fired to invoke the pipeline run. This may be slightly later than the trigger's scheduled time. This expression can only be resolved in schedule scenario.\n",
+    "| `addDays('<timestamp>', <days>, '<format>'?)`    | Add a number of days to a timestamp.\n",
+    "| `addHours('<timestamp>', <hours>, '<format>'?)`  | Add a number of hours to a timestamp.\n",
+    "| `addMinutes('<timestamp>', <minutes>, '<format>'?)`  | Add a number of minutes to a timestamp.\n",
+    "| `addSeconds('<timestamp>', <seconds>, '<format>'?)`  | Add a number of seconds to a timestamp.\n",
+    "| `addToTime('<timestamp>', <interval>, '<timeUnit>', '<format>'?)`   | Add a number of time units to a timestamp.\n",
+    "| `convertFromUtc('<timestamp>', '<destinationTimeZone>', '<format>'?)`  | Convert a timestamp from Universal Time Coordinated (UTC) to the target time zone.\n",
+    "| `convertTimeZone('<timestamp>', '<sourceTimeZone>', '<destinationTimeZone>', '<format>'?)` | Convert a timestamp from the source time zone to the target time zone.\n",
+    "| `convertToUtc('<timestamp>', '<sourceTimeZone>', '<format>'?)` | Convert a timestamp from the source time zone to Universal Time Coordinated (UTC).\n",
+    "| `dayOfMonth('<timestamp>')`   | Return the day of the month component from a timestamp.\n",
+    "| `dayOfWeek('<timestamp>')`    | Return the day of the week component from a timestamp.\n",
+    "| `dayOfYear('<timestamp>')`    | Return the day of the year component from a timestamp.\n",
+    "| `formatDateTime('<timestamp>', '<format>'?)` | Return the date from a timestamp.\n",
+    "| `startOfDay('<timestamp>', '<format>'?)`     | Return the start of the day for a timestamp.\n",
+    "| `startOfHour('<timestamp>', '<format>'?)`   | Return the start of the hour for a timestamp.\n",
+    "| `startOfMonth('<timestamp>', '<format>'?)`  | Return the start of the month for a timestamp.\n",
+    "| `subtractFromTime('<timestamp>', <interval>, '<timeUnit>', '<format>'?)` | Subtract a number of time units from a timestamp.\n",
+    "| `utcNow('<format>'?)` | Return the current timestamp as a string.\n",
+    "| `createdBy()` | Return a user's full name or a service principal's app ID as a string.\n",
+    "| `runId()` | Return run ID as a string of the top-level pipeline run. For nested subgraph runs, this is the run ID of the top-level pipeline run.  For non-nested runs, this is equal to pipeline run ID.\n",
+    "\n",
+    "<font size=2>Note: `?` means optional.</font>  \n",
```

Comment:
Let's move this to the end of "See below table for supported built-in functions in schedule." And change like below.
Note: suffix `?` means optional parameter, otherwise required parameter.